src/docfx/build/page/PageBuilder.cs

```diff
@@ -134,6 +134,7 @@ public void Build(ErrorBuilder errors, FilePath file)
 
             var systemMetadataJObject = JsonUtility.ToJObject(systemMetadata);
 
+            outputModel["schema"] = mime.Value;
```

Comment:
Also consider adding `schema` to each entry in `.publish.json`

Reply:
Where is this requirement from?
Not related to the current feature.

Reply:
No, this isn't required

Reply:
It shows up in `.publish.json` already by [Extensiondata](https://github.com/dotnet/docfx/blob/a3751d30b15eea5565167bb3dfc1ede7691cf268/src/docfx/publish/PublishItem.cs#L36)